lib/ffi.dart

```diff
@@ -21,7 +21,10 @@ void setLang(lang) => _bindings.setSelectLang(lang);
 
 String websocketAddr() => _bindings.websocketAddr().cast<Utf8>().toDartString();
 
-void ffiExit() => _bindings.exitApp();
+void ffiExit() {
+  _bindings.exitApp();
+  SystemChannels.platform.invokeMethod('SystemNavigator.pop');
```

Comment:
@atavism I think we should use both, Go will end all go process and flutter will end flutter process

Reply:
Sounds good. Thanks!